game-core/src/test/java/games/strategy/triplea/delegate/battle/steps/change/MarkNoMovementLeftTest.java

```diff
@@ -71,7 +73,12 @@ void nonAirWithMovementLeftAreMarkedAsMoved() {
 
     markNoMovementLeft.execute(executionStack, delegateBridge);
 
-    verify(delegateBridge).addChange(any(Change.class));
+    verify(delegateBridge)
+        .addChange(
+            argThat(
+                CompositeChangeMatcher.compositeChangeContains(
+                    ObjectPropertyChangeMatcher.propertyChange(
+                        "alreadyMoved", BigDecimal.ONE, BigDecimal.ZERO))));
```

Comment:
nit, would you mind extracting 'altreadyMoved' to a constant? Is that possible? The idea there being the test will automatically update with the constant and it makes it clear as well where that value is coming from for any maintainers that are potentially looking at a failing test.

Second nit, by convention tests use static imports for most everything matcher related (I don't always like that one, but it is a relatively consistent convention)